Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -1030,15 +1030,16 @@
           bounds: "-0.15,-0.3,0.15,0.3"
         hard: false
         mask:
-        - Impassable
-        - BulletImpassable
+       # - Impassable 
+       # - BulletImpassable
+        - Opaque # Delta-V changed from impassable so that they can go through windows
       fly-by: *flybyfixture
   - type: Ammo
   - type: Projectile
     impactEffect: BulletImpactEffectOrangeDisabler
     damage:
       types:
-        Heat: 13
+        Heat: 10
```

Comment:
wait fuck comment this with delta v

almost did a fuck

Reply:
just put down "reduced damage for balance" or some shit B)

Reply:
Thanks for catching that lol